src/core/server/core_app/core_app.ts

```diff
@@ -52,17 +52,6 @@ export class CoreApp {
     router.get({ path: '/core', validate: false }, async (context, req, res) =>
       res.ok({ body: { version: '0.0.1' } })
     );
-
-    coreSetup.savedObjects.registerType({
```

Comment:
@rudolf can it create any problem during migration? Should we delete this SO?

Reply:
As long as there's no dependency on the type, removing a registered type is safe even if an index has existing objects of that type. I'm not aware of any dependency on this either.